src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+            {
+                name: 'MEILISEARCH_ENDPOINT',
+                description: `The host URL of the Meilisearch server. <a class="u-bold" target="_blank" href="https://www.meilisearch.com/docs/learn/getting_started/quick_start/">Learn More</a>`,
+                placeholder: 'http://127.0.0.1:7700',
+                required: true
+            },
+            {
+                name: 'MEILISEARCH_ADMIN_API_KEY',
+                description: `The admin API key for Meilisearch. <a class="u-bold" target="_blank" href="https://docs.meilisearch.com/reference/api/keys/">Learn More</a>`,
+                placeholder: 'masterKey1234',
+                required: true
+            },
+            {
+                name: 'MEILISEARCH_SEARCH_API_KEY',
+                description: `API Key for Meilisearch search operations. <a class="u-bold" target="_blank" href="https://www.algolia.com/doc/guides/security/api-keys/">Learn More</a>`,
+                placeholder: 'searchKey1234',
+                required: true
+            },
+            {
+                name: 'MEILISEARCH_INDEX_NAME',
+                description: `Name of the Meilisearch index to which the documents will be synchronized. <a class="u-bold" target="_blank" href="https://www.meilisearch.com/docs/learn/core_concepts/indexes/">Learn More</a>`,
+                placeholder: 'appwrite_index',
+                required: true
+            },
+            {
+                name: 'APPWRITE_ENDPOINT',
+                description: `The URL endpoint of the Appwrite server. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: 'https://cloud.appwrite.io/v1',
+                placeholder: 'https://cloud.appwrite.io/v1',
+                required: false
+            }
+        ]
+    },
+    {
+        icon: 'icon-vonage',
+        id: 'whatsapp-with-vonage',
+        name: 'WhatsApp with Vonage',
+        tagline: 'Simple bot to answer WhatsApp messages.',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/whatsapp-with-vonage'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/whatsapp-with-vonage) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'VONAGE_API_KEY',
+                description: `API Key to use the Vonage API. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/204014493-How-do-I-find-my-Voice-API-key-and-API-secret-">Learn More</a>`,
```

Comment:
Please fix the `Learn more`, needs to be sentence case.

Also missing periods in every description after `Learn more</a>.